modules/feincms-pages/scripts/init.js

```diff
@@ -7,8 +7,7 @@
         'feincms-pages.page-group',
         'feincms-pages.load-page',
         'feincms-pages.constants',
-        'feincms-pages.routes',
-        'feincms-pages.templates'
+        'feincms-pages.routes'
```

Comment:
Oh, don't they? Every module's `init` needs to register `.templates` as a dependency though, otherwise it will only be loaded if, in an `app.js`, the following is added:
```js
'feincms-pages',
'feincms-pages.templates'
```
I don't remember our `app.js` files containing `.templates` for every module that has them.